packages/devtools_app/lib/src/screens/network/network_screen.dart

```diff
@@ -383,12 +381,24 @@ class ActionsColumn extends ColumnData<NetworkRequest>
     return '';
   }
 
-  List<PopupMenuItem> _buildOptions(NetworkRequest data) {
-    return [
-      if (data is DartIOHttpRequestData) ...[
-        PopupMenuItem(
+  @override
+  Widget build(
+    BuildContext context,
+    NetworkRequest data, {
+    bool isRowSelected = false,
+    bool isRowHovered = false,
+    VoidCallback? onPressed,
+  }) {
+    // Only show the actions button when there are options and the row is
+    // currently selected.
+    if (data is! DartIOHttpRequestData) return const SizedBox.shrink();
+    if (!isRowSelected) return const SizedBox.shrink();
```

Comment:
can we combine these with an OR statement? Also, nit: can we update the comment to include both conditions?

Reply:
It made the code longer, but I can combine them :D